library.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Softwire",
+    "version": "2.0.9",
+    "description": "SoftWire is a software I2C implementation for Arduino and other Wiring-type environments.",
+    "keywords": "I2C",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/stevemarple/SoftWire"
+    },
+    "license": "GNU",
```

Comment:
Looking at https://docs.platformio.org/en/latest/manifests/library-json/fields/license.html, and following the link to https://spdx.org/licenses/, I think the value for `"license"` should be `"LGPL-2.1-or-later"`.